opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -553,6 +557,16 @@ def set_attribute(self, key: str, value: types.AttributeValue) -> None:
             with self._lock:
                 self.attributes[key] = value
 
+    def get_attribute(self, key: str) -> types.AttributeValue:
+        if not key:
```

Comment:
[Always](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) use `is None` or `is not None` to check if something is `None` or not.